aip/general/0148.md

```diff
@@ -135,8 +155,11 @@ Before 2023-07, `purge_time` for soft-deleted resources was also called
 
 ## Changelog
 
+- **2023-07-13**: Introduce the term `annotations` from AIP-128.
```

Comment:
should be today's date I believe, 2023-08-14

Reply:
good catch! fixed.